Include/cpython/unicodeobject.h

```diff
@@ -416,7 +416,7 @@ enum PyUnicode_Kind {
 
 
 /* Fast check to determine whether an object is ready. Equivalent to
-   PyUnicode_IS_COMPACT(op) || ((PyUnicodeObject*)(op))->data.any) */
+   PyUnicode_IS_COMPACT(op) || (((PyUnicodeObject*)(op))->data.any) */
```

Comment:
I think one of the following would be better:
```suggestion
   PyUnicode_IS_COMPACT(op) || ((PyUnicodeObject*)(op))->data.any */
```
or
```suggestion
   (PyUnicode_IS_COMPACT(op) || ((PyUnicodeObject*)(op))->data.any) */
```

Reply:
I was wondering if the outer pair is needed.  I agree that either suggestion is better.